blogpost/Blogpost-draft.md

```diff
@@ -1,39 +1,268 @@
-# Jenkins 2.0 Pipelines
-## Subheading with even more buzzwords
+# JENKINS 2.0 (PART 2) - PIPELINES IN ACTION 
+## How to manage your delivery pipeline as code
 
-2-3 sentence abstract
-* Local CD environment vagrant, ansible
-* Pre-configured Jenkins Pipeline
-* Starting point to play with pipelines in production-like scenario
+----
 
-### Intro / Local environment
-* Short explanation of local sandbox infrastructure
-* Maybe a small/simple diagram showing VMs + Networking + App
-* Links to repos
+_Read our thoughts and experiences we made with Jenkins Pipelines in this blog post, or directly checkout our [github repository](https://github.com/comsysto/jenkins2-pipeline-lab) and see it in action._
 
-### Continuous Delivery scenario concept
-* Short explanation of the CD scenario we build
-* Maybe visualisation of stages + steps
+---- 
 
-### Jenkins pipeline + Jenkins file explained
 
+### Jenkins Pipelines and Jenkinsfiles
+<a name=jenkinsPipelines/> 
+In the last blogpost about Jenkins 2.0 **[LINK]**, which focused on the configuration and administration of Jenkins 2.0, 
+we gave a quick overview of the new Pipline plugin and its features. 
+Now we want to have a closer look at the capabilities of the Pipeline plugin and work out the major innovations and differences that are introduced by the plugin. 
 
+#### Freestyle Jobs vs. Pipelines
+<a name=freestyleVsPipelines/> 
+
+The conventional way of building applications with Jenkins is to define a series of sequential executable steps, each configured as a freestyle job, and then trigger one
+after the other until the whole build process is completed. The chaining of freestyle jobs is usually done via the Jenkins UI, which makes changes and reconfigurations of 
+the job chain rather inconvenient. But fear not, the Jenkins Pipeline plugin is here for the rescue! 
+With the new plugin it becomes possible to orchestrate all workflow steps from commit to deployment within a single job. 
+This job is defined using the Pipeline DSL (domain specific language), which is based on Groovy. 
+Let's have a quick look at the vocabulary of the Pipeline DSL: in the following we have a listing of the three most important terms when it comes to writing pipeline scripts.
+
+* **Stage**: stages are used to break your pipeline into logically distinct sections, which can include one or more build steps. All steps defined within a stage are then visualized as a unique segment in the Pipeline UI.
+* **Step**: build steps are the single executable tasks, which can be chained together in sequence to form stages.
+* **Node**: nodes are wrappers for resource-intensive steps, which should be scheduled and executed in a separate workspace, in order to avoid negative impacts on the overall pipeline performance.
```

Comment:
Node is now mandatory, so it should not sound as if it is optional. Maybe something like:
> Nodes are wrappers for everything that should be scheduled and executed in a separate workspace. Multiple **node** definitions may be used for resource-intensive steps which should be scheduled and executed in a separate workspace, in order to avoid negative impacts on the overall pipeline performance.